### ISSUE TITLE
Roboto downloaded font weights are in 5 families, inconsistent with web.

### ISSUE THREAD

**Issue by estaub** [google/fonts#1928]
When installed locally on MacOS, the full Roboto family (_not_ Condensed, Slab or Mono) is installed as 5 separate font families: 
* Roboto (includes Bold)
* Roboto Black
* Roboto Light
* Roboto Medium
* Roboto Thin

![Screen Shot 2019-04-09 at 8 48 31 PM](https://user-images.githubusercontent.com/1131334/55843948-ddc43480-5b08-11e9-9159-9ec6b4be05a7.png)

As one would expect, a CSS reference to e.g. 100-weight Roboto, fails.  It has to say 100-weight Roboto _Thin_.  This may account for some (but not most) of the problems people have reported with locally-installed Roboto.

Contrast this with e.g. 

![Screen Shot 2019-04-09 at 8 50 46 PM](https://user-images.githubusercontent.com/1131334/55843998-3e537180-5b09-11e9-92f0-b18314db4bcd.png)

**Comment by thlinard** [comment]
Confirmed: platformID 1, nameID 1 for Roboto-Thin.ttf has "Roboto Thin", which is wrong, obviously.

Maybe the whole platformID 1 could be deleted? What FonBakery does in this case?

**Comment by mcmillyonaire** [comment]
I'm experiencing the same issue across all google fonts that have more than just regular + bold weights.

**Comment by tehsayha** [comment]
Yep. Same happening here, even when installed using Skyfonts. Had to go back and dig up an old download zip.

**Comment by laerm0** [comment]
This appears to be fixed now. I will close in anticipation of nobody here still seeing this issue, but will gladly reopen if so.

**Comment by thlinard** [comment]
Hi @laerm0 
The value for nameID 1, platformID 1 is still wrong. Please reopen.
